### ISSUE TITLE
'Launch a worker' button couldn't launch engine tournament

### ISSUE THREAD

**Issue by huydhoang** [lukasmonk/lucaschessR#8]
Version: 0.35a
OS: Windows 10
`The Launch a worker` button inside `Tournament between engines` didn't work.
When I clicked it, a small window brief opened and then crashed. I couldn't even see its content.
And there's no error message, no leftover working process.
Does anyone has an idea what went wrong?

**Comment by lukasmonk** [comment]
There must be at least one game in the queue, in this case a message indicating this would be necessary.

**Comment by huydhoang** [comment]
> There must be at least one game in the queue, in this case a message indicating this would be necessary.
> 

Can you elaborate? I created the tournament and added 10 engines.
More specifically, what are the correct steps to run a tournament and is there any external dependencies required to run it?

**Comment by lukasmonk** [comment]
I have made a video of a test with the R version:
https://drive.google.com/file/d/1TxNDI8Y7jyGmoNkfsiN7dp_13e--BOua/view?usp=sharing

**Comment by huydhoang** [comment]
> I have made a video of a test with the R version:
> https://drive.google.com/file/d/1TxNDI8Y7jyGmoNkfsiN7dp_13e--BOua/view?usp=sharing

Before seeing this video, I saw the new release 0.35b and was just trying it out.
It just works. The user experience is so much better with an instructional message box telling me to create games.
At this point, I think I might have created games in v.11 without knowing that I did.
I am sorry that I made a judgement about the feature so quickly without questioning my process first.
Can I share this video on YouTube as a guide to running tournament with LucasChess or will you do it yourself?

There has never been a better chess training interface to me than LucasChess. Thank you a zillion times!
